src/containers/settings/data/utilities.js

```diff
@@ -0,0 +1,103 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2016, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+import _ from 'lodash';
+
+import * as datetime from '../../../utils/datetime';
+import * as format from '../../../utils/format';
+
+export const DISPLAY_PRESCION_PLACES = 3;
+export const MGDL_UNITS = 'mg/dL';
+export const MMOLL_UNITS = 'mmol/L';
+
+function noData(val) {
+  return val === null || (typeof val === 'undefined');
+}
+
+export function getFormattedTime(startTime) {
```

Comment:
I'm not a fan of this approach adding another layer of wrapping a function that's just in another directory of the same repo. Over time, this will make it harder to read components and understand which functions are being used across multiple components and may very easily lead to duplicate code. In the trends work, I've been [importing the specific functions I need](https://github.com/tidepool-org/viz/blob/jebeck/cgm-trends-iteration.2/src/components/trends/cbg/FocusedCBGSliceHTMLLabels.js#L146) in that component from the datetime utilities.